server/src/main/java/org/opensearch/rest/action/cat/RestShardsAction.java

```diff
@@ -225,6 +225,18 @@ protected Table getTableWithHeader(final RestRequest request) {
             "alias:scti,searchScrollTime;default:false;text-align:right;desc:time scroll contexts held open"
         );
         table.addCell("search.scroll_total", "alias:scto,searchScrollTotal;default:false;text-align:right;desc:completed scroll contexts");
+        table.addCell(
+            "search.point_in_time_current",
+            "alias:spc,searchPointInTimeCurrent;default:false;text-align:right;desc:open point in time contexts"
+        );
+        table.addCell(
+            "search.point_in_time_time",
+            "alias:spti,searchPointInTimeTime;default:false;text-align:right;desc:time point in time contexts held open"
+        );
```

Comment:
nit: call it `search.point_in_time_total_time`

Reply:
Updated